src/components/TextField/TextField.less

```diff
@@ -18,8 +18,6 @@
 
 	&:focus {
 		border: @size-borderWidthFocused solid @color-primary;
-		padding-left: @size-S;
-		padding-right: @size-S;
```

Comment:
the default TextField already uses this for padding, so i removed it here so that we could overwrite it in SearchField

Reply:
Does it cause the text to jump around in the input when focus is applied?

Reply:
nope it doesn't

Reply:
Oh yeah, you're right. Cool looks good.